pkg/noderesourcetopology/filter_test.go

```diff
@@ -586,6 +586,31 @@ func TestNodeResourceTopologyMultiContainerPodScope(t *testing.T) {
 	}
 }
 
+const (
+	running = "Running"
+	pending = "Pending"
+)
+
+// should be filled by the user
+type testUserEntry struct {
+	// description contains the test type and tier as described in TEST.md
+	// and a short description of the test itself
+	description  string
+	initCntCount int
```

Comment:
can we infer safely the count from `len(initCntReq)`? 
IOW, when do we have `initCntCount != len(initCntReq)`?
same question for app containers

Reply:
> can we infer safely the count from len(initCntReq)?

We do, but think of it from an eye of a user, in case you want to know how many init containers/containers this specific test is going to run. 

If you don't have this number in front of your eyes, you'll need to count how many resource objects you have in each of ```initCntReq``` or ```cntReq``` struct, which is quite confusing and inconvenient.

Reply:
OTOH we are setting the same info twice, which is also annoying. I think the inconvenience of counting can be greatly mitigated with better code formatting, making obvious at glance how many container the test is using without having it to tell it explicitely (again)

Reply:
Done